wasp/src/main/java/com/orhanobut/wasp/VolleyNetworkStack.java

```diff
@@ -46,9 +46,9 @@ private RequestQueue getRequestQueue() {
 
     private void addToQueue(final WaspRequest waspRequest, final CallBack callBack) {
         final String url = waspRequest.getUrl();
-        int method = getMethod(waspRequest.getMethod());
-        VolleyListener listener = VolleyListener.newInstance(callBack, url);
-        Request request = new VolleyRequest(method, url, waspRequest.getBody(), listener) {
+        final int method = getMethod(waspRequest.getMethod());
```

Comment:
I would suggest to remove final here as well. Since we use the right away in the next line.